src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -302,6 +302,10 @@ export default function formReducer(state = initialState, action) {
         ? FACILITY_SORT_METHODS.distanceFromResidential
         : FACILITY_SORT_METHODS.alphabetical;
 
+      facilities = facilities.filter(
+        facility => !!facility.address?.city || !!facility.address?.state,
```

Comment:
nit: shouldn't this be an and condition? i.e. only retain facilities where both city and states are available?

Reply:
We don't want to display a facility if we're unable to determine the city or state.